app/products/calls/connection/websocket_client.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {EventEmitter} from 'events';
```

Comment:
any reason to use `EvenEmitter` instead `DeviceEventEmitter` not that matters, just want to be sure this was done with intention

Reply:
This client was ported from web code. If the interface/behaviour of `EvenEmitter` and `DeviceEventEmitter` is the same we can switch. Whatever is the best practice on this side essentially.

Reply:
Yep, I'll do as you suggest -- let me know if we should switch to `DeviceEventEmitter`. 👍

Reply:
yes, let's switch to `DeviceEventEmitter` then, thanks.